requirements.txt

```diff
@@ -1,6 +1,6 @@
-django>=1.11,<2.2
 django-leaflet>=0.23,<0.25
-channels>=2.1.3,<2.2.0
-openwisp-utils>=0.2,<0.3.0
-Pillow>=4.3.0,<6.0
+channels>=2.2.0,<2.3.0
+Pillow>=4.3.0,<6.2
 openwisp-utils>=0.2.1,<0.4.0
+pyasn1==0.4.8
```

Comment:
shouldn't be this installed automatically?

Reply:
It should but it isn't